modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/checkbox/CheckboxDDMFormFieldTemplateContextContributorTest.java

```diff
@@ -58,6 +60,7 @@
 	public void setUp() throws Exception {
 		super.setUp();
 
+		setUpJSONFactoryUtil();
```

Comment:
@marcelabc, do you think we can move the `setUpJSONFactoryUtil()` to the method `setUp` of the class `BaseDDMFormFieldTypeSettingsTestCase` to prevent code repeating in the classes of this commit?